firebase-firestore/src/main/java/com/google/firebase/firestore/remote/OnlineStateTracker.java

```diff
@@ -143,8 +143,8 @@ void handleWatchStreamFailure(Status status) {
         logClientOfflineWarningIfNecessary(
             String.format(
                 Locale.ENGLISH,
-                "Connection failed %d times. Most recent error: %s",
-                MAX_WATCH_STREAM_FAILURES,
+                "Backend didn't respond within %d seconds. Most recent error: %s\n",
+                ONLINE_STATE_TIMEOUT_MS / 1000,
```

Comment:
Is this a stale change?  Else I don't understand it at all.

Reply:
I thought the original error message did not give the user much, since it would just say that the "Connection failed 1 times" along with the most recent error. Saying that the backend didn't respond for 10 seconds, tells the user more about why the state is being set to offline.

Reply:
But that's not why the state is being set to offline...

If the 10-second timer elapses, then we give them an error message saying so here: https://github.com/firebase/firebase-android-sdk/blob/a5dba09385851ed0dea304cb77564dad0b8d64f3/firebase-firestore/src/main/java/com/google/firebase/firestore/remote/OnlineStateTracker.java#L114

But _here_ we are setting the state to offline because our connection failed and so we are giving them the failure error message.  The 10-second timer has _not_ elapsed at this point.

Reply:
This was marked resolved without seeming to take action on the concern expressed here.

Reply:
I believe this was already changed back to the original error message in my last commit.